scripts/common.py

```diff
@@ -325,7 +325,14 @@ def ensure_release_scripts_folder_exists_and_is_up_to_date():
                 'git@github.com:oppia/release-scripts.git'])
 
     with CD(release_scripts_dirpath):
+        ask_user_to_confirm(
+            'Please make sure that the release-scripts folder is clean and '
```

Comment:
`folder is clean` sounds like the folder is empty (is it just me?) Should we call `repo/(git repo) is clean`? Also, should we use the path ../release-scripts instead of just release-scripts so that the user knows where to look for!

Reply:
Makes sense, Done.